server/plugins/TestUGens.cpp

```diff
@@ -44,16 +44,18 @@ struct CheckBadValues : public Unit
 	int		prevclass;
 };
 
+struct Sanitize : public Unit { };
+
 // declare unit generator functions
-extern "C"
```

Comment:
Hmm I'd love to know why (although I do believe you that it works!) - @timblechmann is ```extern "C"``` specified elsewhere? I thought we need that for un-mangled names?